### PR TITLE
PLAT-6496 Update tf-aws-eks mod var schema

### PR DESCRIPTION
**Pull request by miguelhar**
[PLAT-6496](https://dominodatalab.atlassian.net/browse/PLAT-6496)

### CODE REVIEW

File: cdk/setup.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
         "aws-cdk.core~=1.187.0",
         "aws-cdk.lambda-layer-awscli~=1.187.0",
         "aws-cdk.lambda-layer-kubectl~=1.187.0",
-        "boto3~=1.26.0",
+        "boto3==1.26.90",
```
Comment (miguelhar) [inline cdk/setup.py:41]: Updated to fix https://github.com/boto/boto3/issues/3648

Comment (Secretions) [reply cdk/setup.py:41]: You can rebase and revert this. I ended up fixing this in a different way that I think is slightly more correct.

**Review by Secretions — APPROVED** [review]
lgtm other than the boto thing

File: cdk/setup.py (hunk 1 above)
Comment (miguelhar) [reply cdk/setup.py]: done

**Pull request merged**